packages/ui/src/radio/RadioField.tsx

```diff
@@ -1,12 +1,31 @@
 import {
-  FormControl,
+  FormControl as FormControlMui,
   FormControlLabel,
   FormControlLabelProps,
-  FormHelperText,
+  FormHelperText as FormHelperTextMui,
   Radio,
   RadioProps,
 } from '@material-ui/core';
 import { forwardRef, ForwardRefExoticComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import { Color } from '../theme/Color';
+
+const FormControl = styled(FormControlMui)`
+  justify-content: center;
+`;
+
+const FormHelperText = styled(FormHelperTextMui)`
+  color: ${Color.Dark300};
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 400;
```

Comment:
@umrzoq-toshkentov Can we use Typography or Text box for text style rules?

Reply:
@timurhamitov it is overrided FormHelperText. I think we do not need any Typography component here

Reply:
@umrzoq-toshkentov I found this [file with overrides](https://github.com/superdispatch/web-ui/blob/e5722f9c54d83c6821e444d729dabaeb57838880/packages/ui/src/text-field/TextFieldOverrides.tsx#L141). Can we override FormHelperText styles from here?
Can you also check other components where you changed paddings and colors? Maybe they also have overrides like that in our system.

Reply:
@umrzoq-toshkentov Also if you choose to use overrides please check if it will affect other components.

Reply:
@timurhamitov I think no because if we change style from file overrides, it can affect other components as well (TextField, etc.). That is why just isolated and overrided with styled component

Reply:
@umrzoq-toshkentov Have you tried to find a solution using styles from the theme?

Reply:
yes I have tried, but it does not work

Reply:
@doniyor2109 Please share your opinion.

Reply:
I am ok with using styled-component for FormHelperTextMui

Reply:
@doniyor2109 Great.
@umrzoq-toshkentov Then let's move on with PR. But if there will be similar styles for other components - let's think how we can make them sharable styles.

Reply:
@timurhamitov okay got it thanks